docs/source/guides/dm.md

```diff
@@ -36,7 +36,7 @@ the source code for additional options (such as how often to report progress and
 how to wait for the workflow to finish before the bluesky plan proceeds).
 
 ```py
-from instrument.plans.dm_plans import dm_kickoff_workflow
+from apstools.plans.dm_plans import dm_kickoff_workflow
```

Comment:
Same problem here.  No such function in [apstools](https://github.com/search?q=repo%3ABCDA-APS%2Fapstools+dm_kickoff_workflow&type=code).